spec/features/idv/steps/phone_step_spec.rb

```diff
@@ -119,6 +119,36 @@
     expect(page).to have_current_path(idv_doc_auth_step_path(step: :welcome))
   end
 
+  shared_examples 'async timed out' do
+    it 'allows resubmitting form' do
+      user = user_with_2fa
+      start_idv_from_sp
+      complete_idv_steps_before_phone_step(user)
+
+      allow(DocumentCaptureSession).to receive(:find_by).and_return(nil)
```

Comment:
> maybe we could just delete all rows here instead of stubbing?

Seems like it'd be an improvement, but in trying this out, this triggers a different behavior, because it seems to kill off the session value and cause a `none` result instead of a `timed_out` result.

https://github.com/18F/identity-idp/blob/e1be6c208bec5d0e7ae71e3e2e9c34441b36a9f8/app/services/idv/phone_step.rb#L22-L25

Another option (probably more ideal anyways) would be to remove the Redis entries loaded here:

https://github.com/18F/identity-idp/blob/e1be6c208bec5d0e7ae71e3e2e9c34441b36a9f8/app/services/idv/phone_step.rb#L27-L28

But that doesn't appear to be directly exposed.

This code was also partly inherited from the verify step specs, where the mock is restored to simulate an eventual success. Not sure we care to have something similar here, though it'd be made difficult if we outright deleted those entries.

Reply:
Ok good point, likely not worth it, probably fine to continue stubbing